app/bundles/CampaignBundle/Views/Campaign/index.html.twig

```diff
@@ -0,0 +1,30 @@
+{% extends 'MauticCoreBundle:Default:content.html.twig' %}
+
+{% block headerTitle %}{% trans %}mautic.campaign.campaigns{% endtrans %}{% endblock %}
+{% block mauticContent %}mauticCampaign{% endblock %}
+
+{% block actions %}
+{{- include(
+    'MauticCoreBundle:Helper:page_actions.html.twig', {
+      item: item,
+      templateButtons: {
+        'new': securityHasEntityAccess(permissions['campaign:campaigns:create']),
+      },
+      routeBase: 'campaign'
+}) -}}
+{% endblock %}
+
+{% block content %}
+<div class="panel panel-default bdr-t-wdh-0">
+    {{ include('MauticCoreBundle:Helper:list_toolbar.html.twig', {
+        'searchValue': searchValue,
+        'searchHelp': 'mautic.core.help.searchcommands',
+        'action': currentRoute,
+        'filters': filters,
+    }) }}
+
+    <div class="page-list">
+      {# Reference the index.html.php file, how does one map the _content here? #}
```

Comment:
Not sure how to best convert this part from the php file here.

Would also be good to include this in the UPGRADE TO TWIG file

Reply:
Can you please elaborate about the issue here?